backend/app/services/project_service.py

```
@@ -0,0 +1,476 @@
+        .all()
+    )
+
+    result = []
+    for pt in project_tasks:
+        # Get task details
+        task = (
+            db.query(TaskResource)
+            .filter(
+                TaskResource.id == pt.task_id,
+                TaskResource.kind == "Task",
+                TaskResource.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `TaskResource.is_active:` for truth checks

Replace with `TaskResource.is_active`

(E712)